app/src/main-process/main.ts

```diff
@@ -532,15 +532,53 @@ app.on('ready', () => {
     }
   })
 
+  /** An event sent by the renderer asking to select all of the window's contents */
+  ipcMain.on('select-all-window-contents', () =>
+    mainWindow?.selectAllWindowContents()
+  )
+
   /**
    * An event sent by the renderer asking whether the Desktop is in the
    * applications folder
    *
-   * Note: This will be undefined when not running on Darwin
+   * Note: This will return null when not running on Darwin
    */
   ipcMain.handle('is-in-application-folder', async () => {
-    return app.isInApplicationsFolder?.()
+    if (app.isInApplicationsFolder === undefined) {
+      return null
+    }
+
+    return app.isInApplicationsFolder()
```

Comment:
We had a comment to this effect at some point but it probably got lost, could we add it back so that we don't accidentally trust the type system and remove the undefined check? I think the shorthand optional chaining is neat but I'll accept either syntax

```suggestion
    // Contrary to what the types tell you the `isInApplicationsFolder` will be undefined
    // when not on macOS
    return app.isInApplicationsFolder?.() ?? null
```